test/ejabberd_tests/tests/mod_blocking_SUITE.erl

```diff
@@ -0,0 +1,641 @@
+%%==============================================================================
```

Comment:
This suite looks quite comprehensive. I don't see it run though.
1. It's not present here: http://mongooseim-ct-results.s3-website-eu-west-1.amazonaws.com/branch/master/1619/1/big/index.html
2. The coverage for mod_blocking is very low: https://coveralls.io/builds/6502277/source?filename=apps%2Fejabberd%2Fsrc%2Fmod_blocking.erl
3. It's not present in `default.spec`

How can we be sure the implementation works as expected? Do we need this suite at all?